docs/config.toml

```diff
@@ -24,13 +24,19 @@ defaultContentLanguage = "any"
     weight = 1
     [languages.node.params]
       icon = "node-js"
+      frameworks = [
```

Comment:
I think it would be cleaner to keep this data in the frameworks folder. e.g. `lang/reference/frameworks/data/data.md`

Reply:
How would you suggest having an arbitrary length data array in one of those files? I couldn't figure out a clean way, which I figure is just a side effect of using md as a data format. There are other data files that can be in toml, yaml, etc. but those are put at the hugo root, and I didn't know if this was worth adding another data storage.

Reply:
The way we do it with some of the other guides is we store it in the front matter: https://github.com/osohq/oso/blob/main/docs/content/python/getting-started/quickstart/data/data.md

Along with an index file in any that marks it as headless: https://github.com/osohq/oso/blob/main/docs/content/any/getting-started/quickstart/data/index.md

It's not idea, because then you're just writing yaml inside markdown.... But to me it would be more intuitive where the data lives relative to the content.

Reply:
Still having trouble here - per [documentation](https://gohugo.io/content-management/page-bundles/) we can't a) have non-page resources nor b) have resources in a nested directory in a list page. Any suggestions?